docs/guides/admin/docs/installation/rpm-el8.md

```diff
@@ -0,0 +1,170 @@
+
+---
+
+> *This installation **is still experimental**.*
+
+---
+
+This guide is based on a RPM software repository available for Red Hat-based Linux distributions provided by Osnabrück
+University. This repository provides preconfigured Opencast installations and all necessary 3rd-Party-Tools. Using this
+method, you do not have to compile the software by yourself.
+
+
+Availability
+------------
+
+Note that it may take some time (usually about two weeks after a new release is out) before the RPMs are available.
+Watch for announcements on [the users list](https://docs.opencast.org/#mailing-lists) or just check which versions are
+available in the repository.
+
+
+Currently Supported
+-------------------
+
+- CentOS 7.x (x86\_64)
+- Red Hat Enterprise Linux 8.x (x86\_64)
+
+> *Other architectures like i386, i686, arm, … are not supported!*
+
+
+Registration
+------------
+
+Before you can start you need to get an account for the repository. You will need the credentials that you got via
+e-mail after the registration to successfully complete this guide.
+
+Get your account on [pkg.opencast.org](https://pkg.opencast.org).
+
+
+Activate Repository
+-------------------
+
+First you have to install the Opencast repository:
+
+    cd /etc/yum.repos.d
+    curl -O https://pkg.opencast.org/opencast-testing.repo \
+       -d os=el -d version=8 -u [YOUR_USERNAME]
+
+You will be asked for your password.
+
+While this installation is experimental, only the testing repository is available for EL8. A stable repository will be
+made available once the installation has been proven to work properly in different environments.
+
+
+Install Apache ActiveMQ
+-----------------------
+
+The Apache ActiveMQ message broker is required by Opencast. It does not necessarily have to be installed on the same
+machine as Opencast but for an all-in-one that is the usual way. ActiveMQ is available from the Opencast RPM repository
+as well and can be installed by running:
+
+    dnf install activemq-dist
+
+A prepared configuration file for ActiveMQ can be found at `/usr/share/opencast/docs/scripts/activemq/activemq.xml`
+*after Opencast itself has been installed* and should replace `/etc/activemq/activemq.xml`. For an all-in-one
+installation the following command should suffice:
+
+    cp /usr/share/opencast/docs/scripts/activemq/activemq.xml /etc/activemq/activemq.xml
+
+ActiveMQ should be started *prior to* Opencast.
+
+More information about how to properly set up ActiveMQ for Opencast can be found in the [message broker configuration
+documentation](../configuration/message-broker.md).
+
+
+Install Opencast
+------------------
+
+### Basic Installation
+
+For a basic installation (All-In-One) just run:
+
+    dnf install opencast<version>-allinone
+
+…where `<version>` is the major version number of the Opencast release you want to install, e.g. `opencast8-allinone`.
+This will install the default distribution of Opencast and all its dependencies.
```

Comment:
...default distribution of Opencast __8__ and all...

Reply:
Alternatively, this paragraph does not match the one in the el7 docs.  I'm not sure which is better.